<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.0.1 (Python 3.9.5 on darwin)" generated="20210529 23:24:46.624" rpa="false" schemaversion="2">
<suite id="s1" name="Suites">
<suite id="s1-s1" name="LoginTest" source="/Users/wisdomvast/PycharmProjects/pythonProject/TestCaseNew/LoginTest.robot">
<test id="s1-s1-t1" name="LoginTest">
<kw name="Open my browser" library="LoginKeyword">
<arg>${SiteUrl}</arg>
<arg>${Browser}</arg>
<kw name="Open Browser">
<arg>${url}</arg>
<arg>${browser}</arg>
<msg timestamp="20210529 23:24:46.743" level="FAIL">No keyword with name 'Open Browser' found. Did you mean:
    LoginKeyword.Open My Browser</msg>
<status status="FAIL" starttime="20210529 23:24:46.743" endtime="20210529 23:24:46.744"/>
</kw>
<kw name="Maximize Browser Window">
<status status="NOT RUN" starttime="20210529 23:24:46.747" endtime="20210529 23:24:46.747"/>
</kw>
<status status="FAIL" starttime="20210529 23:24:46.737" endtime="20210529 23:24:46.747"/>
</kw>
<kw name="Enter username" library="LoginKeyword">
<arg>${usr}</arg>
<status status="NOT RUN" starttime="20210529 23:24:46.748" endtime="20210529 23:24:46.748"/>
</kw>
<kw name="Enter password" library="LoginKeyword">
<arg>${pwd}</arg>
<status status="NOT RUN" starttime="20210529 23:24:46.748" endtime="20210529 23:24:46.748"/>
</kw>
<kw name="Click login btn" library="LoginKeyword">
<status status="NOT RUN" starttime="20210529 23:24:46.748" endtime="20210529 23:24:46.748"/>
</kw>
<kw name="Verify successful login" library="LoginKeyword">
<status status="NOT RUN" starttime="20210529 23:24:46.748" endtime="20210529 23:24:46.749"/>
</kw>
<kw name="Close Browser" library="LoginKeyword">
<status status="NOT RUN" starttime="20210529 23:24:46.749" endtime="20210529 23:24:46.749"/>
</kw>
<status status="FAIL" starttime="20210529 23:24:46.733" endtime="20210529 23:24:46.750">No keyword with name 'Open Browser' found. Did you mean:
    LoginKeyword.Open My Browser</status>
</test>
<status status="FAIL" starttime="20210529 23:24:46.702" endtime="20210529 23:24:46.753"/>
</suite>
<status status="FAIL" starttime="20210529 23:24:46.626" endtime="20210529 23:24:46.757"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Suites">Suites</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1" name="LoginTest">Suites.LoginTest</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210529 23:24:46.711" level="ERROR">Error in file '/Users/wisdomvast/PycharmProjects/pythonProject/TestCaseNew/LoginTest.robot' on line 2: Importing library 'SeleniumLibrary' failed: ModuleNotFoundError: No module named 'SeleniumLibrary'
Traceback (most recent call last):
  None
PYTHONPATH:
  /usr/local/bin
  /usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python39.zip
  /usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9
  /usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/lib-dynload
  /Users/wisdomvast/Library/Python/3.9/lib/python/site-packages
  /usr/local/lib/python3.9/site-packages</msg>
<msg timestamp="20210529 23:24:46.726" level="ERROR">Error in file '/Users/wisdomvast/PycharmProjects/pythonProject/resources/LoginKeyword.robot' on line 3: Importing library 'SeleniumLibrary' failed: ModuleNotFoundError: No module named 'SeleniumLibrary'
Traceback (most recent call last):
  None
PYTHONPATH:
  /usr/local/bin
  /usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python39.zip
  /usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9
  /usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/lib-dynload
  /Users/wisdomvast/Library/Python/3.9/lib/python/site-packages
  /usr/local/lib/python3.9/site-packages</msg>
</errors>
</robot>
